.github/workflows/update-as.yml

```diff
@@ -35,14 +35,21 @@ jobs:
       run: |
         cd test-fixtures/
         [ ! -e newest_tag.txt ] || rm newest_tag.txt
+    - name: Script to check if branch exists to not commit again
+      run: |-
+        branch=$(curl -X GET -s -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/mozilla-mobile/firefox-ios/branches?per_page=100 | jq -r '.[].name | select(contains("update-cartfile-new-as-tag-${{ env.version }}"))')
```

Comment:
I just compared it with similar curl method we have in NewXcodeBuild workflow:
`resp=$(curl -X GET -s -H 'Accept: application/vnd.github.v3+json' -H "authorization: Bearer ${GITHUB_ACCESS_TOKEN}" https://api.github.com/repos/mozilla-mobile/firefox-ios/commits\?sha\=main\&since\=$YESTERDAY | jq -r '.[].commit.message | select(contains("Auto Update Bitrise.YML"))')`

They look the same in terms of quotes, I think we are fine...